### PR TITLE
Add a very simple introductory page to the pattern library

### PR DESCRIPTION
**Pull request by lyzadanger**
Per popular request, a _very_ simple introductory page for the pattern library (updates to the landing page).

This, of course, wants expansion. But it's a start.

The typography and rhythm look terrible, but we really need to get on a typography system...

### CODE REVIEW

File: styles/pattern-library.scss [hunk 1]
```diff
@@ -235,6 +235,11 @@ $-library-vertical-spacing: 7;
     font-size: 1.75em;
     font-weight: bold;
   }
+
+  &__subheading {
+    font-size: 1.5em;
+    font-style: italic;
+  }
```
Comment (lyzadanger) [inline styles/pattern-library.scss:242]: I hope we can start in on a typography design system ASAP.

File: src/pattern-library/components/LibraryHome.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      <p>
+        <strong>Patterns</strong> are modular implementations of the design
+        system <strong>foundations</strong>—from the atomic to the complex.
+        These implementations are in HTML and CSS.
+      </p>
+
+      <h2 className="LibraryPage__subheading">Components</h2>
+
+      <p>
+        <strong>Components</strong> are preact components that are built using
```
Comment (esanzgar) [inline src/pattern-library/components/LibraryHome.js:32]:
```suggestion
        <strong>Components</strong> are Preact components that are built using
```

**Review by esanzgar — APPROVED** [review]
👍🏼

File: src/pattern-library/components/LibraryHome.js (hunk 2 above)
Comment (lyzadanger) [reply src/pattern-library/components/LibraryHome.js:32]: Interesting, thank you! I had been under the mistaken impression all along that _Preact_ was not capitalized.

**Pull request merged**